upload_stage_rpms

```diff
@@ -2,7 +2,11 @@
 
 . settings
 
-USER='yumrepostage'
-HOST='web01.osuosl.theforeman.org'
+USER='rpmrepostage'
+HOST='rpm.theforeman.org'
```

Comment:
You need to list the host explicitly. `rpm.theforeman.org` will be the CDN and you can't SSH into that.